addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/NetworkUtils.java

```diff
@@ -167,4 +167,25 @@ public static boolean nativePing(String hostname, int port, int timeout)
         return exitValue == 0;
     }
 
+    public static boolean arping(String hostname, int timeout, String networkInterface)
+            throws InvalidConfigurationException, IOException, InterruptedException {
+        Process proc;
+        if (SystemUtils.IS_OS_UNIX) {
+            proc = new ProcessBuilder("arping", "-I", networkInterface, "-w", String.valueOf(timeout / 1000), "-c", "1",
```

Comment:
No, and I don't know how. I think that if it isn't installed ProcessBuilder.start() will throw an IOException? Any suggestions?